### ISSUE TITLE
Webpack fails when linked tns-core-modules

### ISSUE THREAD

**Issue by vchimev** [NativeScript/nativescript-dev-webpack#861]
**Environment**
 - CLI: 5.3.0
 - tns-core-modules: 5.3.0
 - nativescript-dev-webpack: 0.21.0

**Describe the bug**

Webpack compilation fails when linked `tns-core-modules`.

**To Reproduce**
```
tns create appTs --ts && cd appTs
npm link tns-core-modules
tns run android|ios --bundle
```

**Errors**
```
ERROR in Entry module not found: Error: Can't resolve 'inspector_modules.js' in '/Users/vchimev/git/NativeScript/templates/appTabTs/app'

ERROR in /Users/vchimev/git/NativeScript/tns-core-modules/application/application.ts
Module build failed (from ../node_modules/ts-loader/index.js):
Error: TypeScript emitted no output for /Users/vchimev/git/NativeScript/tns-core-modules/application/application.ts.
    at makeSourceMapAndFinish (/Users/vchimev/git/NativeScript/templates/appTabTs/node_modules/ts-loader/dist/index.js:78:15)
    at successLoader (/Users/vchimev/git/NativeScript/templates/appTabTs/node_modules/ts-loader/dist/index.js:68:9)
    at Object.loader (/Users/vchimev/git/NativeScript/templates/appTabTs/node_modules/ts-loader/dist/index.js:22:12)
 @ ./app.ts 38:0-52 39:0-7

ERROR in /Users/vchimev/git/NativeScript/tns-core-modules/text/formatted-string.ts
Module build failed (from ../node_modules/ts-loader/index.js):
Error: TypeScript emitted no output for /Users/vchimev/git/NativeScript/tns-core-modules/text/formatted-string.ts.
    at makeSourceMapAndFinish (/Users/vchimev/git/NativeScript/templates/appTabTs/node_modules/ts-loader/dist/index.js:78:15)
    at successLoader (/Users/vchimev/git/NativeScript/templates/appTabTs/node_modules/ts-loader/dist/index.js:68:9)
    at Object.loader (/Users/vchimev/git/NativeScript/templates/appTabTs/node_modules/ts-loader/dist/index.js:22:12)
 @ /Users/vchimev/git/NativeScript/tns-core-modules/bundle-entry-points.ts 5:72-104
 @ ./app.ts

ERROR in /Users/vchimev/git/NativeScript/tns-core-modules/timer/timer.ts
Module build failed (from ../node_modules/ts-loader/index.js):
Error: TypeScript emitted no output for /Users/vchimev/git/NativeScript/tns-core-modules/timer/timer.ts.
    at makeSourceMapAndFinish (/Users/vchimev/git/NativeScript/templates/appTabTs/node_modules/ts-loader/dist/index.js:78:15)
    at successLoader (/Users/vchimev/git/NativeScript/templates/appTabTs/node_modules/ts-loader/dist/index.js:68:9)
    at Object.loader (/Users/vchimev/git/NativeScript/templates/appTabTs/node_modules/ts-loader/dist/index.js:22:12)
 @ /Users/vchimev/git/NativeScript/tns-core-modules/globals/globals.ts 83:52-68 117:24-40
 @ /Users/vchimev/git/NativeScript/tns-core-modules/bundle-entry-points.ts
 @ ./app.ts

...
```

**Comment by vakrilov** [comment]
I've just tested this scenario with the next versions of `nativescript-dev-webpack` and NS CLI and there are no build errors. 
Seems the issue is resolved or was fixed with configuration.

**Comment by qkdreyer** [comment]
Shouldn't this be fixed for >=5.3, <6 versions ?
